cumulusci/tasks/bulkdata/tests/test_upsert.py

```diff
@@ -0,0 +1,400 @@
+import re
+from unittest import mock
+
+import pytest
+import responses
+
+from cumulusci.tasks.bulkdata import LoadData
+from cumulusci.tasks.bulkdata.step import DataApi, DataOperationStatus
+from cumulusci.tests.util import mock_describe_calls
+
+current_sf_version = "52.0"
```

Comment:
Can these tests use the centralized constant instead of its own, different version?